.changeset/pretty-needles-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-explore': minor
```

Comment:
Just `patch` is enough! There are no breaking changes (apart from the visual behavior I guess).

Reply:
oops, Alright, on my way to change it